packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/appendOrUpdate.operation.ts

```diff
@@ -397,17 +397,17 @@ export async function execute(
 			newColumns.clear();
 		}
 
-		const preparedData = await sheet.prepareDataForUpdateOrUpsert(
-			data,
-			columnsToMatchOn[0],
+		const preparedData = await sheet.prepareDataForUpdateOrUpsert({
+			inputData: data,
+			indexKey: columnsToMatchOn[0],
```

Comment:
same as before could you add names for the rest of the parameters to make the code more consistent and readable?